src/react/databrowser/objects/__tests__/Metadata.test.jsx

```diff
@@ -0,0 +1,101 @@
+import * as s3objects from '../../../actions/s3object';
+import { AddButton, SubButton } from '../../../ui-elements/EditableKeyValue';
+import Metadata from '../details/Metadata';
+import { OBJECT_METADATA } from '../../../actions/__tests__/utils/testUtil';
+import React from 'react';
+import { reduxMount } from '../../../utils/test';
+
+describe('Metadata', () => {
+    const putObjectMetadataMock = jest.spyOn(s3objects, 'putObjectMetadata').mockReturnValue({ type: '' });
+    const optionLabels = ['cache-control', 'content-disposition', 'content-encoding', 'content-language', 'content-type', 'expires', 'website-redirect-location', 'x-amz-meta'];
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    const tests = [
+        'Metadata should render',
+        'should render by default a SelectBox with empty values in each input when there are no key/value present',
+        'should add new key/value metadata and should trigger function if save button is pressed',
+        'remove button and add button should be disabled',
+    ];
+
+    tests.forEach((t, index) => {
```

Comment:
Why do you need a forEach here? 
Debugging and isolating tests can be harder when tests are set through a loop. 
I would prefer the old fashion way, in this case.

Reply:
done